app/main.py

```diff
@@ -1 +1,54 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
```

Comment:
Defining `skills` as a class attribute means it is shared across all instances of `SoftwareEngineer`. This can lead to unexpected behavior if one instance modifies the list. Consider initializing `skills` as an instance attribute in the `__init__` method.